packages/@ember/application/lib/application.ts

```diff
@@ -43,11 +43,11 @@ import type { SimpleDocument, SimpleElement } from '@simple-dom/interface';
   initialized.
 
   ```app/app.js
-  const App = Application.extend({
+  export default class App extends Application {
     ready() {
       // your code here
     }
-  })
+  }
   ```
 
   Because `Application` ultimately inherits from `Ember.Namespace`, any classes
```

Comment:
@tniezurawski if you are interested in doing more edits for this class, I feel like most of this `Ember.Namespace` and "`Application` as a container that holds the other classes in your application" stuff should be deleted.

It was referring to to [globals resolver](https://github.com/emberjs/rfcs/pull/331) which is deprecated/removed at this point. I thought we had deprecated `Ember.Namespace` as well but apparently we didn't. In any case, it doesn't really do anything useful at this point so we don't need to mention it here.

Reply:
The "Event Delegation" stuff probably should go as well though it requires slightly more thoughts on what to do with the content (straight up delete it, or moving it somewhere else, etc).

Most idiomatic Ember apps uses `{{on}}` which doesn't go through event delegation at all and is equivalent to `addEventListener` directly on the element. The event delegation stuff is still used by classic components (`@ember/components`) so it's not completely gone, per se, but definitely doesn't need to be mentioned so front and center here, and setting it largely has no effect at all for a lot of modern apps.

Maybe we can keep it as a `@property` level documentation, or maybe it's more appropriate to talk about this in details in the docs for `@ember/component`.

cc @emberjs/learning-team-managers